homeassistant/components/sensor/steam_online.py

```diff
@@ -59,6 +60,13 @@ def update(self):
         }.get(self._profile.status, 'Offline')
 
     @property
+    def device_state_attributes(self):
+        """Returns the state attributes."""
+        if self._game == None:
+            self._game = 'None'
```

Comment:
This method should not update attributes. That is what the update method is for. Also since the data is already contained in the profile variable that you cache there is no need to store it in an extra variable. Just check profile  here and return the appropriate value